modules/common/src/test/scala/notebok/front/widgets/magic/AnyPointTest.scala

```diff
@@ -0,0 +1,37 @@
+package notebok.front.widgets.magic
+
+import java.text.SimpleDateFormat
+import java.util.GregorianCalendar
+
+import notebook.front.widgets.magic.AnyPoint
+import org.scalatest.FunSuite
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class AnyPointTest extends FunSuite with TableDrivenPropertyChecks {
+
+  private val javaUtilDate = new GregorianCalendar(2017, 9, 19, 12, 30, 33).getTime
+  private val javaSqlTimestamp = new java.sql.Timestamp(javaUtilDate.getTime)
+  private val javaSqlDate = new java.sql.Date(javaUtilDate.getTime)
+
+  private val localTimeZone = new SimpleDateFormat("Z").format(javaUtilDate)
+
+  private val expected = s"Thu Oct 19 12:30:33 2017 $localTimeZone"
+
+  test("Date like value is converted to format expected by java script") {
+    val testData = Table[Any, Seq[String]](
+      ("in", "out")
+      ,(javaUtilDate, Seq(expected))
+      ,(javaSqlTimestamp, Seq(expected))
+      ,(javaSqlDate, Seq(expected))
+      ,(ClassWithDates(javaUtilDate, javaSqlTimestamp, javaSqlDate), Seq.fill(3)(expected))
+    )
```

Comment:
P.S. I'd expect it formatted like this:

```scala
    val testData = Table[Any, Seq[String]](
      ("in", "out"),
      (javaUtilDate, Seq(expected)),
      (javaSqlTimestamp, Seq(expected)),
      (javaSqlDate, Seq(expected)),
      (ClassWithDates(javaUtilDate, javaSqlTimestamp, javaSqlDate), Seq.fill(3)(expected))
    )
```

Reply:
If this is a violation of code style you have then we can reformat it. But let me explain why I use my formatting style. I use `TableDrivenPropertyChecks` extensively in my work. I see it as a good practice to start a line with comma. In "your" format, a very last line is different to others because there is no comma at the end. That way you need to:
* Append comma to last line before adding new test case
* Edit comma placement when you rearrange test cases order
* Edit commas when you comment some cases out temporarily
* 

All these are because If I am a record, "my comma" is on different line. Putting comma at the beginning makes these operations (and git diff) simple. Moreover I haven't discovered any downside to it so I prefer this style. So I prefer this but I am not a boss here :)

Reply:
yeah, diffability that's a scala issue.
P.S. well, it doesn't matter much. but we don't use this elsewhere... (see Databricks or Scala style guides).

Reply:
My opinion is that general style guides makes sense in a situation where you try to enforce it through style checking tool throughout a code base. Then it makes sense to trade minor advantages like this one for ability to check code style programmatically. AFAIK there is no such precaution in place here so I would vote for keeping "my" style here. Should you decide to enforce code style (because of community growth for example) I am fine with refactoring this.